test/parallel/test-timers-setInterval-excludes-callback-duration.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+
+const sleep = function(ms) {
```

Comment:
The linter will want this to be either an arrow function or else a function declaration. So either:

```
const sleep = (ms) => {
```

...or...

```
function sleep(ms) {
```